src/procedures/move-data-disk.html

```diff
@@ -0,0 +1,44 @@
+---
+tags:
+  - add-sdd-existing-installation
+order: 10
+procedures:
+  - title: Configuring Home Assistant and moving your data
+    steps:
+      - title: Go to the storage configuration page
+        image: ha-config-storage.png
+        content: |
+          <ul>
+            <li>
+              Open your Home Assistant instance and navigate to the storage configuration page using the button below:<br><br>
+              <a href="https://my.home-assistant.io/redirect/storage/" target="_blank"><img
+                src="https://my.home-assistant.io/badges/storage.svg"
+                alt="Open your Home Assistant instance and show storage information."
+                style="width: initial"
+              /></a>
+              <br><br>
+              Alternatively, you can go to the storage page via <b>Configuration>System>Storage</b>.
+            </li>
+          </ul>
+      - title: Open the <b>Move to Datadisk</b> dialog
+        image: ha-config-storage-overflow-menu.png
+        content: |
+          <ul>
+            <li>
+              Open the overflow menu by clicking on
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" style="width: 24px"><path d="M12,16A2,2 0 0,1 14,18A2,2 0 0,1 12,20A2,2 0 0,1 10,18A2,2 0 0,1 12,16M12,10A2,2 0 0,1 14,12A2,2 0 0,1 12,14A2,2 0 0,1 10,12A2,2 0 0,1 12,10M12,4A2,2 0 0,1 14,6A2,2 0 0,1 12,8A2,2 0 0,1 10,6A2,2 0 0,1 12,4Z" /></svg>
+              in the top-right of the screen.
+            </li>
+            <li>Select the <b>Move datadisk</b> option</li>
+          </ul>
+      - title: Moving to datadisk
+        image: select-new-datadisk.png
+        content: |
+          <ul>
+            <li>Select your hard drive and click on <b>MOVE</b>.</li>
+            <li class="info">This can take a long time. Please be patient.
```

Comment:
I guess 4 minutes is not that long.

Reply:
A couple of releases ago I optimized it to only copy the effective files (before it was raw dd afaik). With that its quite fast most of the time.

Maybe reorder, first bring the 4 minute example,, and then this point and soften a bit "fairly long time"?